airflow-core/src/airflow/ui/src/i18n/README.md

```diff
@@ -17,8 +17,212 @@
+     - They have the support of an Apache Airflow committer who will act as a translation sponsor.
+- Translation owners candidates MUST go through the approval process detailed in section 6.1.
+
+### 5.3. Approval of new translations
+
+To accept a new translation to the codebase, it MUST be approved through a review process discussed in section 6.2.
+
+### 5.4. Resolution of translation conflicts
+
+Translation conflicts MUST be resolved according to the procedures outlined in section 6.3.
+
+### 5.5. Adding / rephrasing terms
+
+- When new terms are added to the default language, all translation owners SHOULD create a follow-up PR to comply with the changes in their assigned language within a reasonable time.
+- When rephrasing terms in the default language, all translation owners SHOULD do the same as above, **if needed**.
+- In busy times with many parallel UI changes it is acceptable to batch changes together, latest prior a release the differences SHOULD be cleared.
+
+> [!NOTE]
+> Tooling for detecting missing translations is available (see Tools & Resources section below).
+
+### 5.6. Deprecating / refactoring terms
+
+When existing terms are deprecated or refactored (key renamed/relocated but value unchanged), **the contributor initiating the change is responsible for updating all language files, and not the translation/code owner**. Automation through Breeze tooling should be used when available.
+
+### 5.7. Approval and merging of translation-related Pull Requests (PRs)
+
+- If the code owner is also a translation owner of the respective translation:
+    - Others' PRs should be approved and merged normally by them.
+    - Their own PRs should be approved and merged normally by another committer, preferably another code owner in their locale (if such exists).
+- Otherwise, if the code owner is sponsored:
+    - They should merge the translation-related PRs only after they are approved by the translation owner.
+- Other committers may review and approve translation-related PRs in any aspects, but they SHOULD NOT merge them without the approval of the translation owner and the consent of the code owner.
+- Before merging a translation-related PR, the translation should be checked for completeness using the provided tools (see section 8).
+
+> [!WARNING]
+> In languages with different word order than English, or in Right-To-Left (RTL) languages, it is important to validate that the changes are properly reflected in the UI.
+> If they are not, please raise a GitHub issue or a PR for fixing it (separately from the translation PR).
+
+### 5.8. Version release
+
+- Requirements for release managers are defined in the [Release Management Policy](../../../../../../dev/README_RELEASE_AIRFLOW.md).
+
+
+## 6. Procedures
+
+### 6.1. Approval process of a translation owner
+
+- The designated code owner should post a thread in the dev. list to request for approval of the translation owner(s) for a supported translation:
```

Comment:
Maybe we should also include sponsors to the thread and await approval from the sponsor in addition to binding votes rather than commenting in the Github?
Just and idea not a blocker

Reply:
In case of sponsors, they are the designated code owners and should run the approval process.
I"ll clarify it in the next ammendment.